plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -192,9 +192,36 @@ function generateAtomDocument() {
 				$pkpDetails->setAttribute('checksumValue', md5_file($packageFile));
 				break;
 		}
-
+		
 		$entry->appendChild($pkpDetails);
 		$atom->appendChild($entry);
+		
+		$license = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'license');
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'openAccessPolicy', $journal->getLocalizedSetting('openAccessPolicy')));
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'licenseURL', $journal->getSetting('licenseURL')));
+		
+		$mode = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'publishingMode');
+		$mode->setAttribute('mode', $journal->getSetting('publishingMode'));
+		switch($journal->getSetting('publishingMode')) {
+			case 0:
+				$mode->nodeValue = 'Open';
+				break;
+			case 1:
+				$mode->nodeValue = 'Subscription';
+				break;
+			case 2:
+				$mode->nodeValue = 'None';
+				break;
+			default:
+				$mode->nodeValue = 'Unknown';
```

Comment:
If I understand correctly, fatalError() will call die(), which will stop the package generation and deposit process. The error will be logged to the scheduledTaskLog file, but won't be shown to a user.